Core/Utilities/CertificateMgmt.py

```diff
@@ -0,0 +1,125 @@
+"""
+
+DIRAC CertificateMgmt module is used to generate the CAs and revoked certificates
+
+Example:
+
+from DIRAC.Core.Utilities import CertificateMgmt
+
+cl = Elasticsearch( self.__url,
+                    timeout = self.__timeout,
+                    use_ssl = True,
+                    verify_certs = True,
+                    ca_certs = CertificateMgmt.generateCAFile() )
+                    
+or 
+
+sslops = dict( certfile = CertificateMgmt.getCert(/WebApp/HTTPS/Cert),
+               keyfile = CertificateMgmt.getCert(/WebApp/HTTPS/Key),
+               cert_reqs = ssl.CERT_OPTIONAL,
+               ca_certs = CertificateMgmt.generateCAFile('/WebApp/HTTPS/Cert'),
+               ssl_version = ssl.PROTOCOL_TLSv1 ) 
+...                                  
+srv = tornado.httpserver.HTTPServer( self.__app, ssl_options = sslops, xheaders = True )
+
+"""
+import os
+import tempfile
+
+from DIRAC.Core.Security import Locations, X509Chain, X509CRL
+from DIRAC import gLogger, gConfig
+
+
+def getCert( specificLocation = None ):
```

Comment:
getCert/Key have analogs in /Core/Security/Locations, no need to have another utility.